src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -118,7 +125,7 @@ export const InstallButtonWrapperBase = (props: InternalProps) => {
           buttonType={getFirefoxButtonType}
           className={className ? `GetFirefoxButton--${className}` : ''}
         />
-        {!isCompatible && installURL ? (
+        {showDownloadLink ? (
           <div className="InstallButtonWrapper-downloadLink">
             <a
               className="InstallButtonWrapper-downloadLink-link"
```

Comment:
This is a nit-pick (especially as it wasn't added here) but the `Link` in `downloadLink` feels superfluous.

Reply:
Yeah, I think we can change the parent to just `InstallButtonWrapper-download`.